homeassistant/components/velbus/quality_scale.yaml

```diff
@@ -7,10 +7,7 @@ rules:
       This integration does not poll.
   brands: done
   common-modules: done
-  config-flow-test-coverage:
-    status: todo
-    comment: |
-      Split test_flow_usb from the test that tests already_configured, test_flow_usb should also assert the unique_id of the entry
+  config-flow-test-coverage: done
```

Comment:
When was this changed?

Reply:
this file should not have been pushed, i just fixed it